scripts/upgrade.sh

```diff
@@ -131,6 +131,7 @@ function main() {
     journald_persistent
     configure_proxy
     configure_no_proxy
+    kurl_install_support_bundle_configmap
```

Comment:
this should be removed - upgrade won't always have kubectl access, and is only called upon prompting from install.sh